component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1063,23 +1064,23 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 						$fieldval = JText::_("JEV_DURATION_FORMAT");
 						$shownsign = false;
 						// whole days!
-						if (stripos($fieldval, "%wd") !== false)
+                        if (stripos($fieldval, "%wd") !== false)
 						{
-							$days = intval($timedelta / (60 * 60 * 24));
+							$days = (int) $timedelta / (60 * 60 * 24);
 							$timedelta -= $days * 60 * 60 * 24;
 
 							if ($timedelta > 3610)
 							{
 								//if more than 1 hour and 10 seconds over a day then round up the day output
-								$days +=1;
+								++$days;
```

Comment:
Yep it is the same, it makes suggestions and this is more understandable to me and better practice :)